2022/24xxx/CVE-2022-24874.json

```diff
@@ -1,17 +1,17 @@
 {
```

Comment:
('Invalid control character at: line 4 column 32 (char 110)',)

Reply:
This was the smart quote that was causing issues.